src/screens/tabs/Profile/screens/Accounts/AddICNS/index.tsx

```diff
@@ -0,0 +1,167 @@
+import { t } from 'i18next';
+import React, {
+  RefObject,
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
+import {
+  ActivityIndicator,
+  Linking,
+  TouchableOpacity,
+  View,
+} from 'react-native';
+import { Modalize } from 'react-native-modalize';
+import { useDispatch, useSelector } from 'react-redux';
+
+import RainbowButton from '@/components/buttons/RainbowButton';
+import {
+  ActionButton,
+  ActionSheet,
+  Header,
+  Image,
+  Modal,
+  Text,
+} from '@/components/common';
+import { Option } from '@/components/common/ActionSheet';
+import Icon from '@/components/icons';
+import { Colors } from '@/constants/theme';
+import { getICNSData, setReverseResolvedName } from '@/redux/slices/keyring';
+import { ICNS_LEARN_MORE_URL, ICNS_LOGO, ICNS_URL } from '@/services/ICNS';
+
+import styles from './styles';
+
+interface Props {
+  modalRef: RefObject<Modalize>;
+}
+
+const none = 'None';
```

Comment:
could we set the none value as ''? and just keep "None" in the label of the option

Reply:
```
const [selectedName, setSelectedName] = useState<string>(
    reverseResolvedName || ''
  );
```

```
const noneOption = {
      id: 0,
      label: "None", ->> Add to Translations
      onPress: () => setSelectedName(''),
      icon: null,
    };
```